packages/design-system/src/themes/common-theme.ts

```diff
@@ -14,11 +14,13 @@ export interface CommonTheme {
     tooltip: 1000;
   };
   spaces: ['0px', '4px', '8px', '12px', '16px', '20px', '24px', '32px', '40px', '48px', '56px', '64px'];
-  borderRadius: '4px';
-  mediaQueries: {
-    tablet: string;
-    mobile: string;
+  breakpoints: {
+    initial: string;
```

Comment:
is it worth using something like

type MediaQueryString = `@media(${'min-width' | 'max-width'}: ${number}px)`;

to be more specific?